compass/validate.py

```diff
@@ -299,6 +299,8 @@ def _compute_norms(da1, da2, quiet, max_l1_norm, max_l2_norm, max_linf_norm,
 
     result = True
     diff = numpy.abs(da1 - da2).values.ravel()
+    # skip entries where one field or both are a fill value
+    diff = diff[numpy.isfinite(diff)]
```

Comment:
@xylar the fill value is not a NAN in the implementation I tested, it is 
```
temperature:_FillValue = 9.96920996838687e+36 ;
```
so I think the `isfinite` command will still operate on those values, since they are not Nans.

Reply:
@mark-petersen, `xarray` converts fill values to `NaN` when it opens the data set.  I will verify this using https://github.com/E3SM-Ocean-Discussion/E3SM/pull/20 but it has *always* been the case for me in the past.  So I still believe this to be the correct implementation.

Reply:
Thanks for the explanation. I understand now.